website/blog/2019-05-19-15-verdaccio-4-release.md

```diff
@@ -0,0 +1,231 @@
+
+Let's take a quick look at the life cycle and development of Verdaccio community:
+
+- **Verdaccio@2.x (Release name: Birth)** -  Focused on stability, code quality, improvement in  architecture of the old [Sinopia](https://github.com/rlidwka/sinopia) project and community development. 
+
+- **Verdaccio@3.x (Release name: Hope)** - Redesigned the user interface in [React](https://reactjs.org/) and introduced the simplicity of the plugins development. The Verdaccio Team and many contributors made the project almost bug free and robust. This was the time the project started to grow and other projects started using it. 
+
+Verdaccio@4.x is coming up with many exciting new CLI commands for package management, Fast and responsive user interface, security upgrades and easy deployments.
+
+Excited?? Yes !!! Let's go !!
+
+## So what's changed? TL;DR
+- [New User Interface](#new-user-interface)
+    - [New search Process](#new-search-process)
+    - [Register Information](#register-information)
+    - [Packages](#packages)
+    - [Detailed Page](#detailed-page)
+    - [Package Sidebar](#package-sidebar)
+- [New Browser Router APIs](#new-browser-router)
+- [Unpublish Role](#unpublish-role)
+- [Disable Gravatar](#disable-gravatar)
+- [New CLI Commands](#new-cli-commands)
+    - [npm star](#npm-star)
+    - [npm profile](#npm-profile)
+- [JWT Token](#jwt-token)
+- [Docker Improvements](#docker-improvements)
+- [Drop Node 6 Support](#drop-node-6-support)
+- [Plugins](#plugins)
+- [Tech Updates](#tech-updates)
+    - [Verdaccio ESLint Config](#verdaccio-eslint-config)
+    - [Verdaccio Babel Preset](#verdaccio-babel-preset)
+    - [Verdaccio UI Plugin](#vedaccio-ui-plugin)
+    - [Meetup & Conferences](#meetup-&-conferences)
+- [Trusted by Many](#trusted-by-many)
+- [New to Verdaccio / FAQ / Contact / Troubleshoot](#new-to-verdaccio-/-faq-/-contact-/-troubleshoot)
+
+## New User Interface
+
+Verdaccio@4.x comes with a new shiny appealing user interface, providing more details to show and easy to navigate. We did major changes in Verdaccio web application and everything is designed from scratch. 
+
+![verdaccio main page](./assets/verdaccio-main-page.png "verdaccio@4.x main page")
+
+
+### New Search Process
+Verdaccio@3.x has a limited search functionality and it was implemented on the browser side. Verdaccio@4.x provides fast and quick search results from the backend.
+
+![alt text](./assets/nsp.gif)
+
+### Register Information 
+The Register information is easily accessible and can be seen by clicking on `information` icon in header.
+
+![alt text](./assets/register-info.gif)
```

Comment:
Set an alternative text. 😱 this is trully big gif (13.4MB), I tried to load several times and cannot watch it 😭 (and I have a good connection)